docs/guides/developer/docs/api/events-api.md

```diff
@@ -151,14 +159,47 @@ __Example__
 
 Creates an event by sending metadata, access control list, processing instructions and files in a [multipart request](http://www.w3.org/Protocols/rfc1341/7_2_Multipart.html).
 
-Multipart Form Parameters  |Type                             | Description
+Multipart Form Parameters  | Type                            | Description
 :--------------------------|:--------------------------------|:-----------
 `acl`                      | [`acl`](types.md#acl)           | A collection of roles with their possible action
 `metadata`                 | [`catalogs`](types.md#catalogs) | Event metadata as Form param
 `presenter`                | [`file`](types.md#file)         | Presenter movie track
 `presentation`             | [`file`](types.md#file)         | Presentation movie track
 `audio`                    | [`file`](types.md#file)         | Audio track
 `processing`               | [`string`](types.md#basic)      | Processing instructions task configuration
+`scheduling`               | [`string`](types.md#basic)      | Scheduling information (version 1.1.0 and higher)
+
+__Scheduling__
+
+Field      | Required | Type                                 | Description
+:----------|:---------|:-------------------------------------|:-----------
+`agent_id` | Yes      | [`string`](types.md#basic)           | The technical identifier of the capture agent
+`start`    | Yes      | [`datetime`](types.md#date-and-time) | The date and time when the recording shall start
+`end`      | Yes\*    | [`datetime`](types.md#date-and-time) | The date and time when the recording shall end
+`duration` | Yes\*    | [`integer`](types.md#basic)          | The duration of the recording in milliseconds
+`rrule`    | No       | [`string`](types.md#string)          | The optional recurrence rule used to create multiple scheduled events
+
+\* Both `end` and `duration` must be specified for creating multiple scheduled events, either `end` or `duration` must
```

Comment:
Documenting the current implementation makes sense I think.  It would also make it clear *why* we're using the fields the way we are.  I can see a naive implementation supplying both `end` and `duration` of a single event (ie start: 1100, end: 1200, duration: 3600 seconds) and then ending up confused with either an error message or invalid scheduling.

Reply:
Ok, will provide more explanation in the documentation.

Reply:
https://github.com/plapadoo/opencast/pull/20 is now more precise considering the use of end vs duration.

I've tested that coinciding start and end would cause a scheduling conflict, added the documentation but removed it again.

Reason:
The scope of the External API is to provide a facility to ask Opencast to create scheduled events and report back whether Opencast did create those or not. In the latter case, the response is supposed to indicate the reason of not creating them (e.g. a scheduling conflict).

If someone would document the scheduling conflict detection that documentation should rather go to the documentation of the Scheduler service.